frontend/src/components/message.tsx

```diff
@@ -18,6 +18,8 @@ export interface Message {
   content: string;
   reasoning?: string;
   time: string;
+  report?: string;
+  filename?: string;
```

Comment:
Might be getting lost, is `filename` used in here?

Reply:
Yes, on the app.tsx file I use it to pass the filename of the file uploaded to the title of the ESG report.